src/main/java/org/openRealmOfStars/utilities/repository/SunRepository.java

```diff
@@ -0,0 +1,53 @@
+package org.openRealmOfStars.utilities.repository;
+
+import org.openRealmOfStars.starMap.Sun;
+import org.openRealmOfStars.utilities.IOUtilities;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+/**
+ *
+ * Open Realm of Stars game project
+ * Copyright (C) 2016  Tuomo Untinen
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see http://www.gnu.org/licenses/
+ *
+ *
+ * Sun repository class
+ *
+ */
+public class SunRepository {
+
+    /**
+     * Save Sun information to DataOutputStream
+     * @param dos DataOutputStream
+     * @throws IOException if there is any problem with DataOutputStream
+     */
+    public void saveSun(final DataOutputStream dos, final Sun sun) throws IOException {
```

Comment:
Utility like static methods are pretty easy to test. You just give the input and check return value. Static method which read or write into stream are more difficult. I think we should a JUnit test that reads saved game from a file. I could do that I have couple of good saved game files.